### ISSUE TITLE
Bad Request

### ISSUE THREAD

**Issue by ciasis** [IAreKyleW00t/rtorrent-rutorrent-docker#3]
Hi,

I just deployed the container with Portainer on my OMV server, but when I want to access the gui, I get the error message:
400 Bad Request
The plain HTTP request was sent to HTTPS port.

How can I fix this?

**Comment by IAreKyleW00t** [comment]
Hi @ciasis ,

Thanks for taking the time to open an issue for this! I don't have any logic in the nginx config that will automatically redirect you to HTTPS; I'm not sure why I didn't add that actually.

If you manually go to the GUI with `https://`, does that work? If so, I can add some logic to the container to ensure you are always redirected.

**Comment by ciasis** [comment]
Hi,
thank you for getting back to me so quickly.
With `https://`, it works. However, I only access the gui from my local network, so I don't need https ;-). So can I disable it?

And what are the default credentials to login? admin:admin doesn't work.

**Comment by IAreKyleW00t** [comment]
Yeah, I have been meaning to change/fix a few things, including HTTPS and basic auth. That should be getting enabled by default so I will look into that as well (it should be admin:admin too). I’ll try to spend sometime later this week on that

There’s actually a number of benefits (including speed) to using HTTPS, so I would suggest looking into it sometime. :)

**Comment by ciasis** [comment]
Ah, I didn't know that. Yeah, if you could add logic that HTTP requests are automatically redirected to HTTPS, that'd be cool.
Take your time, mate :)
